routes/feeds.js

```diff
@@ -80,6 +80,12 @@ router.post(
   BodyValidationMiddleware.blockAnonymousPostV2,
   feed.blockAnonymousPostV2
 );
+router.post(
+  '/block-anonymous-comment-v2',
+  isAuthV2,
+  BodyValidationMiddleware.blockAnonymousCommentV2,
+  feed.blockAnonymousCommentV2
+);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new route `/block-anonymous-comment-v2` is correctly defined with the necessary middleware and handler functions. However, it's important to ensure that the `isAuthV2` middleware properly authenticates the user before proceeding to the `BodyValidationMiddleware.blockAnonymousCommentV2` and `feed.blockAnonymousCommentV2` functions. Also, make sure that error handling is properly implemented in these middleware and handler functions to handle any potential errors or exceptions.
```diff
  router.post(
    '/block-anonymous-comment-v2',
+   // Ensure proper authentication
    isAuthV2,
+   // Validate request body
    BodyValidationMiddleware.blockAnonymousCommentV2,
+   // Handle blocking of anonymous comments
    feed.blockAnonymousCommentV2
  );
```
Please note that the diff block above does not suggest code changes but rather highlights areas to pay attention to in the implementation of the respective functions.


<!-- This is an auto-generated comment by OSS CodeRabbit -->